packages/api/es/collections.js

```diff
@@ -8,9 +8,9 @@ const { BaseSearch } = require('./search');
 const ES_MAX_AGG = 2147483647;
 
 class Collection extends BaseSearch {
-  constructor(event, type, index) {
+  constructor(event, type, index, includeStats = true) {
```

Comment:
Should it default to false?

Reply:
I didn't think so, because I didn't look to see how often it was being used before.  I did make the endpoints default to false.  I'll take a look.

Reply:
Looks like it's only used in the create-reconciliation-report lambda and some tests.  So I can make it default to false

Reply:
ef9077e
(and we don't actually need the stats in reconciliation report, so I left it alone.)